release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -186,11 +187,11 @@ The Network Observability Operator releases updates independently from the {prod
 
 [id="workload-partitioning-enhancement"]
 ==== Workload partitioning enhancement
-With this release, platform pods deployed with a workload annotation that includes both CPU limits and CPU requests will have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In prior releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook. 
+With this release, platform pods deployed with a workload annotation that includes both CPU limits and CPU requests will have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In prior releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook.
```

Comment:
```suggestion
With this release, platform pods deployed with a workload annotation that includes both CPU limits, and CPU requests will have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In prior releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook.
```

Reply:
Unless I am misreading the sentence, I think that without the comma is correct. That said, I sort of lose the thread in the middle of the sentence.
```suggestion
With this release, CPU limits are accurately calculated and applied as a CPU quota for platform pods deployed with a workload annotation that includes both CPU limits and CPU requests. In earlier releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook.
```
Does this change the meaning?
Also, see the[ ISG guidance for _earlier_ ](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#earlier) as the preferred term for older releases.